bm-persona/Common/FilterTableView/FilterTableViewCell.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  ResourceTableViewCell.swift
+//  bm-persona
+//
+//  Created by Anna Gao on 11/6/19.
+//  Copyright © 2019 RJ Pimentel. All rights reserved.
+//
+
+import UIKit
+
+class FilterTableViewCell: UITableViewCell {
+    
+    static let kCellIdentifier = "filterCell"
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        contentView.frame = contentView.frame.inset(by: UIEdgeInsets(top:0, left:5, bottom:10, right:5))
+        
+        // add shadow on cell
+        backgroundColor = .clear
+        contentView.layer.masksToBounds = false
+        contentView.layer.shadowOpacity = 0.25
+        contentView.layer.shadowRadius = 5
+        contentView.layer.shadowOffset = .zero
+        contentView.layer.shadowColor = UIColor.black.cgColor
+        contentView.layer.shadowPath = UIBezierPath(rect: contentView.bounds.insetBy(dx: 4, dy: 4)).cgPath
+        // add corner radius on `contentView`
+        contentView.backgroundColor = .white
```

Comment:
I don't believe you can override contentView to be a CardView because it's inherently a part of all TableViewCells. Could be wrong, but I think you can only add subviews to contentView.

Reply:
Yea, you're right. Let's keep it the way it is then.